imod_coupler/drivers/dfm_metamod/msw_wrapper.py

```diff
@@ -88,53 +88,33 @@ def get_surfacewater_ponding_allocation(self) -> NDArray[np.float_]:
         """
         return self.get_value_ptr("ts2dfmput")
 
-    def set_surfacewater_ponding_allocation(
-        self, ponding_allocation: NDArray[np.float_]
-    ) -> None:
-        """sets ponding volume allocation in metaswap
+    def get_surfacewater_ponding_realised(self) -> NDArray[np.float_]:
+        """gets ponding volume allocation pointer in metaSWAP
```

Comment:
what is "ponding realised" ?  is it a water height? a flux? how is it different form " ponding" (so without the realised specifier?)

Reply:
Its a water volume per dtsw-timestep. MetaSWAP uses 2 different variables for allocation and realization phase of ponding water. So we have to communicate with two different pointers unfortunately.

Reply:
can you add a comment in the code with this text? it would help